Basic Rules.txt

```diff
@@ -369,7 +369,31 @@ level 15. After level 15, the limit for stat points is 20. The number of
 initial base stat points that you have to apply is listed with your Race or 
 Species. See chapter 1: Races if you haven't yet. 
 
-Leveling Up:
+==== Assigning A Class to your Character ====
+	Most Characters start at level 1 so the Player will choose a Class for that 
+Character at that time. A Character will only ever have one Class at a time and
+under normal circumstances cannot change their class once it is picked. 
+	On occasion especially for new Players, a character may start at level 0 
+with no Class. This is usually for a tutorial mission and a Class is selected

```

Comment:
Missing a comma before "and." The comma is necessary because you are separating two complete sentences with a coordinating conjunction.

Reply:
wilco